extensions/amp-access/0.1/access-expr-impl.jison

```diff
@@ -0,0 +1,130 @@
+
```

Comment:
Cool. You may want to consider just putting it into the default compilation tasks.

Also, why not remove the generated JS now?